tephra-hbase-compat-0.96/src/main/java/co/cask/tephra/hbase96/coprocessor/TransactionProcessor.java

```diff
@@ -149,10 +149,13 @@ public void preGetOp(ObserverContext<RegionCoprocessorEnvironment> e, Get get, L
     throws IOException {
     Transaction tx = getFromOperation(get);
     if (tx != null) {
+      boolean excludeCurrentWritePtr = isAttributeSet(get, TxConstants.TX_EXCLUDE_CURRENT_WRITE);
```

Comment:
@abaranau other visibility constraints - such as excluding uncommitted write versions  - are also attributes of the transaction, not the individual request. I am simply suggesting to follow the existing model. Code is more maintainable if we do similar things in similar ways.